### ISSUE TITLE
Error on install

### ISSUE THREAD

**Issue by kelto** [elastic/ansible-elasticsearch#176]
Hi ! I'm trying to use ansible to install elasticsearch but always stumble on this error:

```
TASK [elasticsearch : Remove shield plugin] ************************************
task path: /home/xxx/ansible-elasticsearch/tasks/xpack/elasticsearch-xpack-install.yml:15
skipping: [Node005] => {
    "changed": false,
    "skip_reason": "Conditional check failed",
    "skipped": true
}

TASK [elasticsearch : Install shield plugin] ***********************************
task path: /home/xxx/ansible-elasticsearch/tasks/xpack/elasticsearch-xpack-install.yml:29
skipping: [Node005] => {
    "changed": false,
    "skip_reason": "Conditional check failed",
    "skipped": true
}

TASK [elasticsearch : set_fact] ************************************************
task path: /home/xxx/ansible-elasticsearch/tasks/xpack/shield/elasticsearch-shield-file.yml:2
skipping: [Node005] => {
    "changed": false,
    "skip_reason": "Conditional check failed",
    "skipped": true
}

TASK [elasticsearch : List Users] **********************************************
task path: /home/xxx/ansible-elasticsearch/tasks/xpack/shield/elasticsearch-shield-file.yml:5
skipping: [Node005] => {
    "changed": false,
    "skip_reason": "Conditional check failed",
    "skipped": true
}

TASK [elasticsearch : set_fact] ************************************************
task path: /home/xxx/ansible-elasticsearch/tasks/xpack/shield/elasticsearch-shield-file.yml:11
skipping: [Node005] => {
    "changed": false,
    "skip_reason": "Conditional check failed",
    "skipped": true
}

TASK [elasticsearch : Remove Users] ********************************************
task path: /home/xxx/ansible-elasticsearch/tasks/xpack/shield/elasticsearch-shield-file.yml:15
fatal: [Node005]: FAILED! => {
    "failed": true,
    "msg": "'users_to_remove' is undefined"
}

```

Here is the configuration i have:

```
es_scripts: false
es_templates: false
es_log_dir: /home/es/logs/
es_scripts_fileglob: /home/es/es_script/
es_templates_fileglob: /home/es/es_templates/
es_version_lock: false
ansible_user: root
```

**Comment by jakommo** [comment]
@kelto can you post the complete playbook, so I can try to reproduce?

I'm currently looking into https://github.com/elastic/ansible-elasticsearch/issues/167 and already have some fixes locally available. Seem like there are a lot of issues with the shield/x-pack stuff.

Looking at your config, do you even want to setup shield/x-pack?

**Comment by kelto** [comment]
Well no i do not need it. I've removed it from the roles, which make it work. 

I'm sorry i can not give the complete playbook, but for more information the server was a debian 8 (jessie)

**Comment by veger** [comment]
I beleive I have fixed this with #179

**Comment by gingerwizard** [comment]
Agreed. Closing.
